CppCoreGuidelines.md

```diff
@@ -1696,7 +1696,7 @@ Use the ISO Concepts TS style of requirements specification. For example:
 
 ##### Note
 
-Soon (maybe in 2017), most compilers will be able to check `requires` clauses once the `//` is removed.
+Soon (maybe in 2018), most compilers will be able to check `requires` clauses once the `//` is removed.
 For now, the concept TS is supported only in GCC 6.1.
```

Comment:
This could be interpreted to mean that gcc 6.4 and 7.2 do not support concepts, which is not true. How about changing it to say "only supported in GCC 6 and later." ?